src/question_baseselect.ts

```diff
@@ -485,7 +504,18 @@ export class QuestionSelectBase extends Question {
     this.choicesByUrl.fromJSON(val.toJSON());
   }
   /**
-   * The list of items. Every item has value and text. If text is empty, the value is rendered. The item text supports markdown.
+   * Gets or sets choice items. This property accepts an array of objects with the following structure:
+   *
+   * ```js
+   * {
+   *   "value": any, // A value to be saved in the survey results
+   *   "text": String, // A display text. This property supports Markdown. When `text` is undefined, `value` is used.
+   *   "imageLink": String // A link to the image or video that represents this choice value. Applies only to Image Picker questions.
+   *   "customProperty": any // Any property that you find useful
```

Comment:
I would add an example how to add a new property, because if the property is not added into "itemvalue" our Serializer will not set it into object from JSON.